change/@fluentui-react-infobutton-b36ec3f1-5811-4e2b-ac9e-306c17c23e2e.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=29418&commitId=3036fd53d7efc6d09b7be1ce27d8d61a0a9cb87b&env=prod&clientType=FLUENTUI)


  <details><summary><b>InfoButton</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|InfoButton.default - High Contrast.active.chromium.png|1426| Changed|
|InfoButton.default - Dark Mode.active.chromium.png|1243| Changed|
|InfoButton.default.active.chromium.png|1749| Changed|
|InfoButton.size.rest.chromium.png|1447| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

Reply:
Expected changes